containers/Makefile

```diff
@@ -68,6 +68,11 @@ postgresql-clean:
 concent-api-clean:
 	docker rmi $(IMAGE_PREFIX)concent-api:$(IMAGE_VERSION) --force
 
+build/concent-api/RELEASE-VERSION: build/nginx-proxy/concent-api-assets.tar
```

Comment:
It's `concent-api-assets.tar` that should depend on `RELEASE-VERSION`, not the other way around. This file should be included inside the archive.